plasmapy/diagnostics/charged_particle_radiography.py

```diff
@@ -162,6 +164,16 @@ def __init__(
         self.mag = 1 + np.linalg.norm(self.detector) / np.linalg.norm(self.source)
         self._log(f"Magnification: {self.mag}")
 
+        # Calculate the size of the grid imaged onto the detector
+        # (used for auto-choosing the size keyword when making histograms)
+        self.grid_scale_length = np.max(
+            [
+                np.max(np.abs(self.grid.pts0.to(u.m).value)),
+                np.max(np.abs(self.grid.pts1.to(u.m).value)),
+                np.max(np.abs(self.grid.pts2.to(u.m).value)),
+            ]
+        )
+
```

Comment:
Make this into a property so has proper documentation and is only calculated when needed.  Right now I do not see why it needs to be part of the class `__init__`.